app/src/main/java/chat/rocket/android/helper/Avatar.java

```diff
@@ -30,10 +30,16 @@
   };
   private final String hostname;
   private final String username;
+  private final String avatarUrl;
 
   public Avatar(String hostname, String username) {
+    this(hostname, username, null);
+  }
+
+  public Avatar(String hostname, String username, String avatarUrl) {
```

Comment:
I think `Avatar` class should be designed just for rendering User's avatar, and it would be better to define rendering Message avatar independent of Avatar.